proto/p4/p4runtime.proto

```diff
@@ -154,8 +154,8 @@ message FieldMatch {
     int32 prefix_len = 2;  // in bits
   }
   message Range {
```

Comment:
Hmm ... for some reason, angle brackets got eaten-up in my comment above.
I mean to say:
... to be of type bit{W} or int{W}
Replace curly-bracket and angle-bracket.